app/views/support/LandingPageOps.scala

```diff
@@ -35,6 +39,24 @@ object LandingPageOps {
     def getIncentiveTermsAndConditions: String = {
       promotion.asIncentive.fold("") { p => p.promotionType.termsAndConditions.mkString }
     }
+    def getIncentiveLegalTerms: String = {
+      promotion.asIncentive.fold("") { p => p.promotionType.legalTerms.mkString }
+    }
   }
 
+  def planToOptions(promoCode: PromoCode, promotion: AnyPromotion)(in: CatalogPlan.Paid): SubscriptionOption = {
+    val planPrice = promotion.asDiscount.fold(in.charges.gbpPrice)(adjustPrice(in, _))
+    val saving = promotion.asDiscount.fold(in.saving)(_ => None)
```

Comment:
if isdefined then None else Some(in.saving) ? so saving happens when there isn't a discount?